command/alloc_exec.go

```diff
@@ -168,7 +173,7 @@ func (l *AllocExecCommand) Run(args []string) int {
 			return 1
 		}
 
-		allocStub, err = getRandomJobAlloc(client, jobID, "", ns)
+		allocStub, err = getRandomJobAlloc(client, jobID, group, ns)
```

Comment:
We had this parameter and just weren't using it?! :facepalm: 

A quick search for other callers says we could add it to the `alloc fs` command too. Want to do that while we're here?

Reply:
Yes! Will do.